cdap-cli/src/main/java/co/cask/cdap/cli/command/security/RevokeActionCommand.java

```diff
@@ -70,13 +70,6 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
   }
 
   @Override
-  public String getPattern() {
-    return String.format("security revoke entity <%s> [%s <%s>] [%s <%s>] [actions <actions>]",
-                         ArgumentName.ENTITY, ArgumentName.PRINCIPAL_TYPE, ArgumentName.PRINCIPAL_TYPE,
-                         ArgumentName.PRINCIPAL_NAME, ArgumentName.PRINCIPAL_NAME);
-  }
-
-  @Override
   public String getDescription() {
     return "Revokes a user's permission to perform certain actions on an entity. <actions> is a comma-separated list.";
```

Comment:
perhaps override the description in the subclassses  so it becomes more precise for each case?